src/main/java/org/jitsi_modified/impl/neomedia/codec/video/vp8/DePacketizer.java

```diff
@@ -485,4 +555,19 @@ public static int getHeight(byte[] buf, int off)
             return (((buf[off + 6] & 0xff) << 8) | buf[off + 5] & 0xff) & 0x3fff;
         }
     }
+
+    /**
+     * Utility - set the value of a bit within a byte.
+     */
+    private static void setBit(byte[] buf, int off, byte mask, boolean set)
```

Comment:
should this be moved to the Byte extensions in jitsi-rtp?

Reply:
Do you think I should I add a `putBit` to `Byte` to set bits by mask, or should I change `vp8.Depacketizer` to set by bitPos?  (It currently gets bits by mask, so if I do the latter I should probably change everything to be consistent.)

Reply:
The mask approach feels more flexible and compact, so my vote goes with that.

Reply:
Done.